template_credentials_test.go

```diff
@@ -0,0 +1,71 @@
+package transloadit
+
+import (
+	"fmt"
+	"testing"
+	"time"
+)
+
+func TestTemplateCredentials(t *testing.T) {
+	t.Parallel()
+
+	client := setup(t)
+	templateCredentialName := generateTemplateName()
+
+	templateCredentialPost := NewTemplateCredential()
+	templateCredentialPost.Name = templateCredentialName
+	templateCredentialPost.Type = "s3"
+	templateCredentialPost.Content["key"] = "xyxy"
+	templateCredentialPost.Content["secret"] = "xyxyxyxy"
+	templateCredentialPost.Content["bucket"] = "mybucket.example.com"
+	templateCredentialPost.Content["bucket_region"] = "us-east-1"
+
+	// Step 1: Create a brand new templateCredentialPost
```

Comment:
What is templateCredentialPost? Should it just say template credential here and in all following comments?